decidim/decidim
# Update the Twitter icons and brand name to X

#### :tophat: What? Why?
We were recently asked to update the Twitter logos to the new X logos which is what this PR does. It also adds the `twitter-x` logo to the legacy `icons.svg` file (ionicon icon set) in order to make this backportable.

The icon added to the legacy icon set and the social share icons is downloaded from:
https://about.twitter.com/en/who-we-are/brand-toolkit

The new brand color (`#000000`) is from the cover image of the company's X (Twitter) profile as there was no brand guidelines available from the above link:
https://twitter.com/x

Note that this will not change the column names or anything else code related referring to the word `twitter`. The purpose of this PR is to update the brand name and its logo as there is more work associated with changing the column names.

#### :pushpin: Related Issues
- Related to #11494

#### Testing
- Rebuild the development app to get the new defaults e.g. for the social share services
  * Alternatively you can apply the changes manually to the `config/secrets.yml` file as in this PR
- Check the footer with the default seeds
- Configure the Twitter omniauth login (you can configure the correct icon from the application secrets, this PR adds it to the generator template but it does not apply to the `development` environment)
- Go to check the conference speakers and open the details modal to see the X handle
- Go to a participatory space (e.g. a process) and check the "process dropdown" i.e. the process related mega menu and see that the process hashtag is with the correct icon

### :camera: Screenshots

![Footer social logos](https://github.com/decidim/decidim/assets/864340/6ad310f8-b359-4d9f-854b-980e272a4a74)

![Omniauth login buttons](https://github.com/decidim/decidim/assets/864340/f239431a-c182-42c0-aaad-b57ac1ddbc54)

![Social share modal](https://github.com/decidim/decidim/assets/864340/1eefaa0b-9248-4ce1-a604-fb7c0daea7d2)

![Conference speaker modal](https://github.com/decidim/decidim/assets/864340/498cf412-3ebb-42e6-bc0b-e6652c5bfff3)

![Participatory space menu dropdown](https://github.com/decidim/decidim/assets/864340/e9bc79d7-8fed-4fd6-8529-621ee11f9c35)
